source/administration/mmctl-cli-tool.rst

```diff
@@ -91,6 +90,34 @@ The ``mmctl`` tool uses ``go`` modules to manage dependencies, so you need to ha
 
   make build
 
+Local mode
+----------
+
+Local mode allows platform administrators with access to the Mattermost server to run mmctl commands against the API without needing to have a user registered. To ensure secure usage of this API, the server exposes a local socket that only a user with access to the server's file system can access. This local socket flags the requests coming from the socket, so they can reach the handlers without requiring a user session. 
```

Comment:
Small changes to avoid repetition:

```suggestion
Local mode allows platform administrators with access to the Mattermost server to run mmctl commands against the API without needing to have a user registered. To ensure secure usage of this API, the server exposes a local socket that only a user with access to the server's file system can access. The requests coming from the socket are treated as authorized, so they can reach the handlers without requiring a user session.
```